chore: added blog -Resolving merge conflict on Github in category,subcategory

devasyarajguru opened the pull request:
<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->

## Fixes Issue #674 

<!-- Remove this section if not applicable -->

<!-- Example: Closes #31 -->

## Changes proposed
Added a blog link to Opensource section with proper Name , description, url , category , subcategory
<!-- List all the proposed changes in your PR -->

## Screenshots

<!-- Add all the screenshots which support your changes -->
<img width="960" alt="Resolving guide" src="https://github.com/rupali-codes/LinksHub/assets/84851506/9e8d9350-78bc-46c3-8b1e-30b0d01268e6">

## Note to reviewers



<!-- Add notes to reviewers if applicable -->
None

CBID2 reviewed (approved):
Great job @devasyarajguru! 😄 LGTM! 😄

devasyarajguru commented:
I have to fix the changes you have told above? I have to fix it again in my local setup(VS Code) and again add, commit, push changes ?? That's it? @CBID2

devasyarajguru commented:
> > I have to fix the changes you have told above? I have to fix it again in my local setup(VS Code) and again add, commit, push changes ?? That's it? @CBID2
> 
> Have you checked the Contributing file before making this PR @devasyarajguru?

Yes I have checked it. Why?

devasyarajguru commented:
Hey @CBID2 I just misunderstand the comment. Actually I thought I have to fix the change but it's already changed by you. I hope it is fine?

CBID2 commented:
> Hey @CBID2 I just misunderstand the comment. Actually I thought I have to fix the change but it's already changed by you. I hope it is fine?

It's fine @devasyarajguru! :) Rupali just needs to approve your pull request.

devasyarajguru commented:
> > Hey @CBID2 I just misunderstand the comment. Actually I thought I have to fix the change but it's already changed by you. I hope it is fine?
> 
> It's fine @devasyarajguru! :) Rupali just needs to approve your pull request.

Thank you very much @CBID2 @rupali-codes . You both are very supportive and active. It was great to contribute! Looking forward to do more!

CBID2 reviewed (changes requested):
Hi @devasyarajguru! :) There are some branch conflicts that need to be solved.

devasyarajguru commented:
For solving the conflicting branch , do I have to perform [git pull upstream main] ?? and then git push?

devasyarajguru commented:
Please checked this. I have resolved the issues @CBID2

CBID2 reviewed (dismissed):
Great job @devasyarajguru!😄 LGTM! 😄

devasyarajguru commented:
> Great job @devasyarajguru!😄 LGTM! 😄

Thanks @CBID2 . When will be my PR merged?

CBID2 commented:
> > Great job @devasyarajguru!😄 LGTM! 😄
> 
> Thanks @CBID2 . When will be my PR merged?

When you get an approved review by @rupali-codes @devasyarajguru! 😊

devasyarajguru commented:
> > > Great job @devasyarajguru!😄 LGTM! 😄
> > 
> > 
> > Thanks @CBID2 . When will be my PR merged?
> 
> When you get an approved review by @rupali-codes @devasyarajguru! 😊

Okay no problem. But just tell me that I have resolved conflicts properly or not?

rupali-codes reviewed (commented):
This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀

devasyarajguru commented:
> This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀

Does that count my contribution? @CBID2 @rupali-codes , and will this PR will remain open? Please explain it as I am a beginner.

devasyarajguru commented:
> > This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀
> 
> Does that count my contribution? @CBID2 @rupali-codes , and will this PR will remain open? Please explain it as I am a beginner.

Also I have done valid contribution should'nt my PR gets merged??

rupali-codes commented:
> > > This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀
> > 
> > 
> > Does that count my contribution? @CBID2 @rupali-codes , and will this PR will remain open? Please explain it as I am a beginner.
> 
> Also I have done valid contribution should'nt my PR gets merged??

Yes your PR is completely valid, but the resource has already been added. Could you please work on some other issues and close this PR? I really appreciate your time and commitment, but we can not have duplicate resources for the same thing.

devasyarajguru commented:
Subject: Disappointment Regarding Unmerged Pull Request

Dear @CBID2 @rupali-codes ,

I hope this message finds you well. Firstly, I want to express my sincere appreciation for the opportunity to contribute to the project and for the efforts you all put into maintaining it. I understand that you receive numerous pull requests on a regular basis, making the task of managing and merging them quite challenging.

I am writing to you today with a sense of disappointment regarding the unmerged pull request (PR) I recently submitted. I had worked diligently on addressing an issue that was assigned to me, and I firmly believed that my proposed solution was both valid and beneficial to the project.

I would like to remind you that the project follows the "First Come First Serve" principle when it comes to assigning and resolving issues. In accordance with this principle, I had made a considerable effort to understand the problem thoroughly, brainstorm a solution, and implement it in a way that adhered to the project's guidelines and best practices. I dedicated significant time and energy to ensure the quality and correctness of the code in my PR.

To my surprise, upon submitting my PR, I discovered that a similar solution had already been merged into the project. I understand that situations like this can arise due to various factors, including oversight or lack of visibility on my part. Nevertheless, I genuinely believed that my contribution would bring value to the project and that it deserved careful consideration.

Once again, I want to express my gratitude for the opportunity to contribute to this project and for your ongoing commitment to its maintenance. I sincerely hope that we can engage in a constructive dialogue regarding my concerns, as I am eager to learn and grow as a member of this community.

Thank you for your time and consideration.

Yours sincerely,

Devasya

rupali-codes commented:
If you don't want to close the PR unmerged, kindly refere replacing the resources with something else.
You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

devasyarajguru commented:
> If you don't want to close the PR unmerged, kindly refer replacing the resources with something else. You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

Ohkay so can I replace this resource in this PR only?

rupali-codes commented:
> > If you don't want to close the PR unmerged, kindly refer replacing the resources with something else. You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> 
> Ohkay so can I replace this resource in this PR only?

yeah

devasyarajguru commented:
> > > If you don't want to close the PR unmerged, kindly refer replacing the resources with something else. You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> > 
> > 
> > Ohkay so can I replace this resource in this PR only?
> 
> yeah, can also do in the issue, so it won't create confusions

Updated the website link as mentioned by @rupali-codes . Also updated issue which closes #674. Title and Description updated of PR #749

rupali-codes reviewed (changes requested):
Hey, just remove the empty line, other than that, looks great to me

CBID2 commented on database/opensource/blogs.json:
```diff
@@ -22,9 +22,9 @@
-      "description": "This blog will help you learn about everything Open Source",
-      "url": "https://opensource.guide/",
+      "name": "Resolving merge conflict on Github",
+      "description": "This blog will help you to resolve merge conflict on Github",
```
[Proofreading]
```suggestion
      "description": "This blog will help you to resolve merge conflicts on GitHub",
```

CBID2 reviewed (changes requested):
Hi @devasyarajguru! :) I pointed out some proofreading errors that could be improved.

rupali-codes commented:
> Hi @devasyarajguru! :) I pointed out some proofreading errors that could be improved.

let's merge it, 😄

rupali-codes commented:
@CBID2 could you approve the PR please :)

devasyarajguru commented:
> Hi @devasyarajguru! :) I pointed out some proofreading errors that could be improved.

What are proofreading errors? I don't know , I am contributing first time. And No problem , it would be great for me to improve this particular thing and then afterwards you can merge my PR. It would be great for me if you help me pointing out my improvements! Thanks! @rupali-codes @CBID2

CBID2 reviewed (approved):
LGTM!

CBID2 commented:
> Hey @CBID2 and @rupali-codes . Thanks for co-operating and supporting me. I want to know is there any improvement to do in this PR? Also @CBID2 I want to make sure that I will improve upon the last comment you gave me about proofreading errors. I want to know is there anything from my side on this now?

You’re welcome @devasyarajguru! We just need @rupali-codes to force push your PR. As far as improving on this pull request, not necessary

devasyarajguru commented:
Thank You very much @rupali-codes and @CBID2 . You guys are very supportive and active. It was great collaborating with you!😊

rupali-codes commented:
> Thank You very much @rupali-codes and @CBID2 . You guys are very supportive and active. It was great collaborating with you!😊

Its great to have you :)
Thanks
